### DISCUSSION TITLE
Neel Parihar - GitHub Externship Proposal

### DISCUSSION THREAD

**Discussion by NeelParihar** [hoppscotch/hoppscotch#1622] [Q&A]
**Application number:  25-04_Nee923_tfa_258**

**Full name: Neel Parihar**

**E-mail: neelparihar599@gmail.com**

**GitHub username: [NeelParihar](https://github.com/NeelParihar)**

**Prior experience:**
- [x] HTML
- [x] CSS
- [ ] SCSS
- [x] Tailwind CSS
- [x] JavaScript
- [ ] TypeScript
- [ ] Vue.js
- [ ] Nuxt.js
- [x] NestJS
- [x] Firebase
- [ ] Prisma
- [ ] GraphQL
- [x] PostgreSQL
- [ ] Go lang

---

**Proposal type:**
- [x] Existing feature request
- [ ] New feature proposal
- [ ] Refactor existing feature

**Proposal title: gRPC-Web support** 

**Proposal description:**  Like swagger, let users view message formats from .proto / URL and test the server methods by sending any messages from the Hoppscotch.

**Comment by AndrewBastin** [comment]
This proposal feels really short. Please expand with an elaborate breakdown of what you are trying to do over the period of Externship.

**Reply by NeelParihar** [reply]
Hey @AndrewBastin , i will not be participating in Github Externship program because of some reasons and thus you can omit this proposal. Thank you for reviewing my proposal really appreciated it.

**Reply by AndrewBastin** [reply]
Oh, that is sad.

Anyways, stay safe and hope to see you next time!
